oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elasticsearch/ElasticsearchIndexProviderService.java

```diff
@@ -140,6 +149,8 @@ private void activate(BundleContext bundleContext, Map<String, ?> config) {
         //initializeExtractedTextCache(config, statisticsProvider);
 
         elasticsearchConnection = getElasticsearchCoordinate(config);
+        indexPrefix = PropertiesUtil.toString(config.get(PROP_INDEX_PREFIX), PROP_INDEX_PREFIX_DEFAULT);
```

Comment:
A general discussion - would it make sense to add a random String to this prefix ? Since we are providing this as a configurable property, technically there could be a case where 2 different tenants configure the same index prefix and we are not handling that here.
@fabriziofortino @thomasmueller